sync_radar_data_globus.py

```diff
@@ -120,14 +113,16 @@ def __init__(self, client_id, client_secret=None, transfer_rt=None):
         parser.add_argument("-t", "--data_type",
                             help="One of {} Default: 'raw'".format(self.possible_data_types),
                             default='raw')
+
         parser.add_argument("sync_local_dir", help="Path on endpoint to sync data to")
         args = parser.parse_args()
+
         self.sync_year = args.sync_year
         self.sync_month = "{:02d}".format(args.sync_month)
         self.sync_pattern = args.sync_pattern
         self.data_type = args.data_type
         self.sync_local_dir = args.sync_local_dir
-
+        self.mirror_root_dir = "~/chroot/sddata"
         self.sanity_check()
 
         # Get a transfer client
```

Comment:
Side note: if you want things to be more clear with formatting strings you can do this:
" {year} and {month} are in the future".format(year=syn_year, month=sync_month)